types/articleTypes.ts

```diff
@@ -0,0 +1,34 @@
+export interface Article {
+  content: string;
+  createdAt: string;
+  id: number;
+  image: string | null;
+  likeCount: number;
+  title: string;
+  updatedAt: string;
+  writer: {
+    id: number;
+    nickname: string;
+  };
+}
+
+export interface ArticleApiData {
+  page?: number;
+  pageSize?: number;
+  orderBy?: "recent" | "like";
+  keyword?: string;
+}
+
+export interface ArticleProps {
+  initialArticles: Article[];
+}
+
+export interface ArticleProp {
+  article: Article;
+}
+
+export enum DeviceTypePageSize {
+  MOBILE_PAGE_SIZE = 1,
+  TABLET_PAGE_SIZE = 2,
+  DESKTOP_PAGE_SIZE = 3,
+}
```

Comment:
이전 리뷰에서 enum을 사용해보고 싶다고 하셨던 기억이 있는데, 아주 잘 활용해 주셨습니다. 👍
다만, 첫번째 값에만 숫자를 지정하면 2, 3에 해당하는 값은 굳이 지정할 필요는 없으니 참고해주세요!
(헷갈릴수 있어서 모든 값에 숫자를 지정하는 경우도 있지만 DeviceType 특성상 아주 많은 값이 들어올 가능성은 없어 비교적 처음 시작값만 지정해도 충분해 보이는 부분입니다.)

```suggestion
export enum DeviceTypePageSize {
  MOBILE_PAGE_SIZE = 1,
  TABLET_PAGE_SIZE // 2의 값을 가짐
  DESKTOP_PAGE_SIZE // 3의 값을 가짐 
}
```

Reply:
넵 enum에서 설정할 값이 연속적이라면 첫 숫자만 값을 지정하면 되니까 편하네요!🫡